tasks/windows/windows.rake

```
@@ -178,6 +178,7 @@ namespace :windows do
 
     # Only copy the .bat files into place
     cp_p(FileList["conf/windows/stage/bin/*.bat"], "stagedir/bin/")
+    FileUtils.cp('downloads/mcollective/ext/windows/mco.bat', 'stagedir/bin/mco.bat') if File.exists?('downloads/mcollective/ext/windows/mco.bat')
```

I'm also :+1: to failing hard here as well for the same reason.  I'll adjust the PR so that we can get this merged today.